zproject/default_settings.py

```diff
@@ -366,3 +366,6 @@
 # users, and you would like to save some disk space. Soft-deactivated
 # returning users would still be caught-up normally.
 AUTO_CATCH_UP_SOFT_DEACTIVATED_USERS = True
+
+# Default Organization Description
+DEFAULT_ORGANIZATION_DESCRIPTION = "The coolest place in the universe"
```

Comment:
For future i18n reasons I don't think we want to make this a setting; instead, we should just check if the database field is "".

Reply:
ok